core-java-modules/core-java-lang-syntax-2/README.md

```diff
@@ -13,3 +13,4 @@ This module contains articles about Java syntax
 - [Introduction to Basic Syntax in Java](https://www.baeldung.com/java-syntax)
 - [Java ‘protected’ Access Modifier](https://www.baeldung.com/java-protected-access-modifier)
 - [[<-- Prev]](/core-java-modules/core-java-lang-syntax)
```

Comment:
TODO: Update the link with that of the published article

Reply:
This will be done after the article goes live, not for you to do. Please revert this file